README.md

```diff
@@ -138,6 +138,8 @@ tables-to-go -help
   -?	shows help and usage
   -d string
     	database name (default "postgres")
+  -f
+        force, skip tables that encounter errors but construct all others
```

Comment:
Argh I was about to merge when I saw you only updated the Readme. Can you please also update the  https://github.com/fraenky8/tables-to-go/blob/master/doc.go#L43 - thanks so much!

Reply:
Sorry, I totally missed that! Added it in d2cc4e4